parm/config/gefs/config.resources

```diff
@@ -41,15 +41,15 @@ case ${machine} in
     ;;
   "AWSPW")
     export PARTITION_BATCH="compute"
-    max_tasks_per_node=36
+    max_tasks_per_node=48
     ;;
   "AZUREPW")
     export PARTITION_BATCH="compute"
-    max_tasks_per_node=24
+    max_tasks_per_node=36
     ;;
   "GOOGLEPW")
     export PARTITION_BATCH="compute"
-    max_tasks_per_node=32
+    max_tasks_per_node=30
```

Comment:
Looking at the table at the top of [Configuring NOAA Cloud Service Providers](https://global-workflow.readthedocs.io/en/latest/noaa_csp.html#configuring-noaa-cloud-service-providers), it looks like these could be
```bash
    export PARTITION_BATCH="compute"
    max_tasks_per_node=72
    ;;
  "AZUREPW")
    export PARTITION_BATCH="compute"
    max_tasks_per_node=48
    ;;
  "GOOGLEPW")
    export PARTITION_BATCH="compute"
    max_tasks_per_node=60
```

That said, there's no harm (except possibly cost) for running fewer tasks/node.  Just want to check.

Reply:
If these are to be updated, then you will likely want to change parm/config/gfs/config.resources as well.

Reply:
The numbers I put there are physical Cores.
If we double the numbers, which means we are using vCPUs, I have not tested yet.
Let me run a test and see how that goes.

Reply:
Ah, I see.  No need to run that test.  The global workflow does not support hyperthreading.

Reply:
Tried c7i.24xlarge, which AWS/PW said it has 96 vCPUs, if I set 
    npe_node_max=96
    max_tasks_per_node=96
    
    Then we get errors:
    
sbatch: error: Batch job submission failed: Requested node configuration is not available
2024-12-04 20:52:02 +0000 :: mgmt-weihuang-awsepicweirocky8c7i24xlarge-00013 :: Submitting gfs_fcst_seg0
2024-12-04 20:52:02 +0000 :: mgmt-weihuang-awsepicweirocky8c7i24xlarge-00013 :: Submission status of gfs_fcst_seg0 is pending at druby://10.1.0.105:45141
2024-12-04 20:53:02 +0000 :: mgmt-weihuang-awsepicweirocky8c7i24xlarge-00013 :: Submission status of previously pending gfs_fcst_seg0 is failure!  sbatch: error: CPU count per node can not be satisfied

Reply:
Noted, thanks.  This should stay as is, then.

Reply:
Those are virtual CPU numbers. GW wants the real Core number, if uses virtual core, GW won't run.